declarations.gms

```diff
@@ -272,163 +272,163 @@ qDummyObj                                     "Define dummy objective function"
 Variables
 
 *' *** Power Generation Variables
-VElecCapChpPla(allCy,CHP,YTIME)	          "Capacity of CHP Plants (GW)"
-VLoadCurveConstr(allCy,YTIME)	          "Parameter for load curve construction (1)"
-VEstBaseLoad(allCy,YTIME)	              "Estimated base load (GW)"	
-VElecDem(allCy,YTIME)                     "Total electricity demand (TWh)"
+VCapElecChp(allCy,CHP,YTIME)	          "Capacity of CHP Plants (GW)"
+VLambda(allCy,YTIME)	          "Parameter for load curve construction (1)"
```

Comment:
There's now a VLambda. as well as a VLamda variable. This may cause errors and confusion.

Reply:
There is only VLambda. The VLamda parameter has been renamed.